docs/egress.md

```diff
@@ -127,6 +128,44 @@ The `externalIPPool` field specifies the name of the `ExternalIPPool` that the
 be assigned to. It can be empty, which means users should assign the `egressIP`
 to one Node manually.
 
+### TrafficShaping
+
+The `bandwidth` field impose rate-limiting on egress traffic of the Egress. `rate`
+specifies the maximum transmission rate. `burst` specifies maximum burst for throttle.
+`rate` and `burst` should follow the k8s Quantity definition
+[here](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/),
+e.g. 300k, 100M, 2G. All backend workloads selected by a rate-limited Egress share the
+same bandwidth while sending egress traffic via this Egress.
```

Comment:
I think we should explicitly state that "if these limits are exceeded, the traffic will be dropped"

Reply:
Added.